Gemfile

```diff
@@ -49,7 +49,9 @@ gem 'chosen-rails' #  jquery multiselect plugin for advanced search
 gem 'devise', '~> 4.7.1'
 gem 'devise-guests'
 gem 'geoblacklight', '~> 2.1'
+gem 'geoblacklight_sidecar_images', git: 'https://github.com/pulibrary/geoblacklight_sidecar_images', branch: 'service_url'
```

Comment:
It looks like this PR was merged, can we just use master? Or even a released version (0.6.4)